practicals/image-pipeline/filter.c

```diff
@@ -43,7 +43,7 @@ void blur5(unsigned char *restrict imgData, unsigned char *restrict out, long w,
   // of non-zero elements in the filter.
   float scale = 1.0 / 35.0;
 
-#pragma acc parallel loop collapse(2) gang vector copyin(imgData[0:w * h * ch]) copyout(out[0:w * h * ch])
+// TO DO: parallelize this loop and add data clauses
```

Comment:
Can you write it `TODO` instead of `TO DO`, just for "compatibility" with the rest exercises?